docs/manual/manual_advanced_arming.rst

```diff
@@ -0,0 +1,756 @@
+----------------------------------
+The arming throttle region is specified by two parameters: *Arm Throttle Upper Limit* and *Arm Throttle Lower Limit*. These two parameters can be found in the IQ Control Center 
+under the General tab, as shown below. Users can configure these to specify any range of throttle commands as the arming throttle region. When a sufficient number of 
+throttle commands in the arming throttle region are received, an arming transition will occur. For more details on the number of arming throttle commands required to 
+trigger an arming transition, see :ref:`arming_consecutive_throttles`. When the arming transition occurs, the module will perform its arming behavior as detailed in :ref:`advanced_arming_behavior`.
+
+.. figure:: ../_static/manual_images/arming/arming_throttle_limits.png
+    :align: center
+    :width: 60%
+    :alt: Arming Throttle Region Limits
+
+    Arming Throttle Region Limits in IQ Control Center
+
+The *Arm Throttle Upper Limit* defines the upper, more positive bound of the arming throttle region. It should always be more positive than or equal to the 
+*Arm Throttle Lower Limit*. The *Arm Throttle Lower Limit* defines the lower bound of the arming throttle region. Every throttle command the module receives in 
+this region is considered an arming throttle and will contribute towards an arming transition. The number of arming throttles required to cause an arming 
+transition is determined by the *Consecutive Arming Throttles to Arm* parameter as discussed in the :ref:`arming_consecutive_throttles` section below.
+
+The image below illustrates where the arming throttle region would be for an example setup where the Arm Throttle Upper Limit is set to 0.3 and the Arm Throttle Lower Limit 
+is set to 0.1. Any throttle command between 10% and 30% is an arming throttle.
+
+.. figure:: ../_static/manual_images/arming/arming_example_arming_throttle_region_1.png
+    :align: center
+    :width: 60%
+    :alt: Example Arming Throttle Region
+
+    Example Arming Throttle Region
+
+**There is a separate parameter that determines whether the arming throttle region exists at all. The** *Arm On Throttle* **parameter, shown below in 
+IQ Control Center, allows users to toggle the arming throttle region on or off completely.** If this parameter is set to *Do Not Arm on Throttle*, 
+then there is no arming throttle region and throttles cannot cause an arming transition. Otherwise, arming throttles can be used as previously described.
+
+.. figure:: ../_static/manual_images/arming/arming_arm_on_throttle_parameter.png
+    :align: center
+    :width: 60%
+    :alt: Arm on Throttle Parameter
+
+    Arm on Throttle Parameter in IQ Control Center
+
+.. _arming_consecutive_throttles:
+
+Consecutive Arming Throttles
+------------------------------
+**Users can also specify how many consecutive arming throttles the module needs to receive before the arming transition occurs.** This can be useful 

```

Comment:
Users can specify the number of consecutively received arming throttles required to cause an arming transition.